charts/stable/home-assistant/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 ---
 apiVersion: v2
-appVersion: 2022.5.4
+appVersion: 2022.8.2
 description: Home Assistant
 name: home-assistant
-version: 13.4.2
+version: 13.5.2
```

Comment:
I think `13.5.0` would be a more common choice.

Reply:
yeah I'm not sure what I was thinking there!